<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlowPairwiseParametersDialog</class>
 <widget class="QDialog" name="SlowPairwiseParametersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slow Pairwise Parameters</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gap opening penalty:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="AgDoubleSpinBox" name="gapOpenPenaltyDoubleSpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="value">
      <double>10.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Gap extension penalty:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AgDoubleSpinBox" name="gapExtensionPenaltyDoubleSpinBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="value">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="checkBox_2">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Weight matrix:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="weightMatrixComboBox_">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QCheckBox" name="checkBox_3">
     <property name="text">
      <string>Use default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AgDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>gui/widgets/AgDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapOpenPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>19</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>toggled(bool)</signal>
   <receiver>gapExtensionPenaltyDoubleSpinBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>51</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SlowPairwiseParametersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SlowPairwiseParametersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>135</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_3</sender>
   <signal>toggled(bool)</signal>
   <receiver>weightMatrixComboBox_</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
